cmd/kubeadm/app/phases/addons/dns/dns.go

```diff
@@ -231,7 +232,7 @@ func createCoreDNSAddon(deploymentBytes, serviceBytes, configBytes []byte, clien
 
 	coreDNSClusterRoles := &rbac.ClusterRole{}
 	if err := kuberuntime.DecodeInto(clientsetscheme.Codecs.UniversalDecoder(), []byte(CoreDNSClusterRole), coreDNSClusterRoles); err != nil {
-		return errors.Wrap(err, "unable to decode CoreDNS clusterroles")
+		return errors.Wrap(err, "%s ClusterRole", UnableToDecodeCoreDNS)
```

Comment:
should be `Wrapf` because we are passing an argument.

Reply:
Thank you, it has been modified.